### PR TITLE
Remove duplicate logging

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

 /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -157,9 +157,6 @@ func runCommand(cmd *cobra.Command, args []string, opts *options.Options) error
 
 // Run executes the scheduler based on the given configuration. It only return on error or when stopCh is closed.
 func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}) error {
-	// To help debugging, immediately log version
-	klog.V(1).Infof("Starting Kubernetes Scheduler version %+v", version.Get())
```
Comment (hzxuzhonghu) [inline cmd/kube-scheduler/app/server.go]: It is logged already at L142

**Review by hex108 — APPROVED** [review]
/lgtm

File: cmd/kube-scheduler/app/server.go (hunk 1 above)
Comment (bsalamat) [reply cmd/kube-scheduler/app/server.go]: L142 is not as complete and does not have verbosity. Could you remove L142 instead, or better replace L142 with this line.

Comment (hzxuzhonghu) [reply cmd/kube-scheduler/app/server.go]: OK

Comment (hzxuzhonghu) [reply cmd/kube-scheduler/app/server.go]: @bsalamat   please add a tag

**Review by draveness — APPROVED** [review]
/lgtm

**Comment by hzxuzhonghu** [comment]
@k82cn would you approve?

**Review by draveness — APPROVED** [review]
/priority backlog
/assign @huang-wei
/lgtm

Thanks

**Review by Huang-Wei — COMMENTED** [review]
/approve

**Pull request merged**